.env.example

```
@@ -0,0 +1,13 @@
+EXPO_PUBLIC_PLACES_API_KEY=
+EXPO_PUBLIC_DIRECTIONS_API_KEY=
+
+DATABASE_URL=
+
+EXPO_PUBLIC_SERVER_URL=http://localhost:3000
+
+EXPO_PUBLIC_SUPABASE_URL=https://supa.tkrtecnologia.net
+EXPO_PUBLIC_SUPABASE_PUBLISHABLE_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.ewogICJyb2xlIjogImFub24iLAogICJpc3MiOiAic3VwYWJhc2UiLAogICJpYXQiOiAxNzE1MDUwODAwLAogICJleHAiOiAxODcyODE3MjAwCn0.vyDeYsp2rsE1pkDBITrbkv6UnDiCswR_cf2CVI8SxuM
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EXPO_PUBLIC_SUPABASE_PUBLISHABLE_KEY key should go before the EXPO_PUBLIC_SUPABASE_URL key

(UnorderedKey)